pages/linux/btrfs-restore.md

```diff
@@ -0,0 +1,24 @@
+# btrfs restore
+
+> Try to salvage files from a damaged btrfs filesystem.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-restore>.
+
+- Restore all files from btrfs filesystem to given directory:
```

Comment:
Perhaps this will read better?
```suggestion
- Restore all files from a btrfs filesystem to a given directory:
```

Reply:
Agree! Committed the suggestions and pushed